---
layout: page
title: Evolving Seas RCN Working Groups
---

## Working groups formed in the 2019 Synthesis Workshop

### Selection Across the Life Cycle: Genome-informed approaches and insights for characterizing targets of selection across complex life cycles in eukaryotes

Molly Albecker, Sam Bashevkin, Matthew Hahn, Matthew Hare, Holly Kindsvater, Stacy Krueger-Hadfield, Katie Lotterhos, Adam Reitzel, Mary Sewell, Laetitia Wilkins

**Overview:** Our goal is to develop and write a perspective article on the use of genome-focused approaches to characterize the mechanisms of selection across the life cycle. We have organized this article by summarizing current studies using single-generation studies paired with genomic analyses, comparing how genomic approaches complement other approaches (life history modeling, quantitative genetics), and listing recommendations for experimental design based on simulation studies. Marine organisms are ideally suited for studies quantifying how selection at different life cycle stages impacts genome-wide variation due to comparatively wide life cycle diversity, large numbers of propagules/offspring, feasibility of family designs in small research spaces, and the availability of genomic and experimental resources.

Product: [Does a complex life cycle affect adaptation to environmental change? Genome-informed insights for characterizing selection across complex life cycle](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2021.2122) Published 2021 in Proceedings of the Royal Society B 

### Fluctuating Selection

Mark Bitter, Scott Burgess, Hans Dam, Groves Dixon, Sarah Donelan, Santiago Herrera, Lee Karp-Boss, Carly Kenkel, Lisa Komoroske, Katie Lotterhos, Kerry Nickols, Emily Rivest, Juliet Wong 

** Goals:** The overarching goal of our working group is to provide a single, synthetic article in which we accomplish the following:
- Provide a novel synthesis of theoretical literature exploring evolutionary dynamics under different regimes of fluctuating selection pressures.
- Highlight attributes of marine systems that make them ideal for testing these theoretical predictions.
- Review empirical studies that have effectively tested theoretical predictions in a marine setting, and highlight those theoretical predictions that still warrant substantial practical corroboration.
- Provide perspective regarding how adaptation to various regimes of contemporary/historic variability may influence adaptation to climate change.

Product: [Fluctuating selection and global change: a synthesis and review on disentangling the roles of climate amplitude, predictability and novelty](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2021.0727) Published 2021 in Proceedings of the Royal Society B 

### Surf and Turf - Comparing intraspecific differences in thermal limits between marine and terrestrial habitats

Matthew Sasaki, Brian Cheng, Sarah Gignoux-Wolfsohn, Cynthia Hays, Morgan Kelly, Seema Sheth, Jordanna Barley, Alysha Putnam, Andrew Villenueve

**Goals:** To explore intra-specific differences in thermal limits in marine, intertidal, freshwater, and terrestrial taxa, and to compare the scaling of those differences between realms, using geographic distance and environmental differences as our moderators. 
We have surveyed the literature on adaptive divergence of upper and lower thermal limits, and compiled a list of papers that meet our requirements for inclusion. We have made significant progress towards extracting data from the complete collection of >60 papers. We have generated code that calculates geographic distance between populations, extracts environmental data for each site from commonly used temperature databases, and analyzes/visualizes the data.

Products: 

[Limited plasticity in thermally tolerant ectotherm populations: evidence for a trade-off](https://doi.org/10.1098/rspb.2021.0765) Published 2021 in Proceedings of the Royal Society B 

Sasaki, M., J.M. Barley, S. Gignoux-Wolfsohn, C.G. Hays, M.W. Kelly, A.B. Putnam, S. N. Sheeth, A.R. Villeneuve, B.S. Cheng. (2022) [Greater evolutionary divergence of thermal limits within marine than terrestrial species](https://www.nature.com/articles/s41558-022-01534-y). Nature Climate Change 12:1175-1180 

Sasaki, M. and B.S. Cheng. (2022) [Populations adapt more to temperature in the ocean than on land](https://www.nature.com/articles/s41558-022-01535-x).
Nature Climate Change 12: 1098-1099. Research Briefing

### Modeling evolution in marine systems

Amanda Xuereb, Peter Tiffin, Huijie Xue, Megan Phifer-Rixey

**Goals:** One challenge of modelling evolution in marine taxa is incorporating the effects of pelagic life stages that impact population structure and connectivity. This group will review the use of eco-evolutionary marine models and discuss the impact of factors such as larval dispersal, larval settlement, and sharp clines in environmental conditions on local adaptation.

Product: [Individual-based eco-evolutionary models for understanding adaptation in changing seas](https://doi.org/10.1098/rspb.2021.2006) Published 2021 in Proceedings of the Royal Society B 


### Predicting Diversity and Divergence in the Sea   	
 
Gideon Bradburd, Reid Brennan, Joanna Kelley, Misha Matz, Jamie Pringle, Cynthia Riginos, Rachel Toczydlowski, John Wares 	

**Goals:** The goal of the working group is to quantify patterns of genetic diversity and divergence in marine organisms and understand the biotic and abiotic processes that drive them.  To accomplish this goal, we are taking a data synthesis approach, aggregating publicly available, georeferenced, next-generation sequence datasets for marine species, as well as biotic trait data for those species and oceanographic data for the areas in which they were sampled.  Taken together, these data will allow us to integrate across scales both temporally (micro- to macroevolution) – and spatially (local to global) to test hypotheses about what biotic and abiotic factors predict genomic diversity and divergence in marine ecosystems.

Products:

Toczydlowski, R. H. et al. Poor data stewardship will hinder global
 genetic diversity surveillance. PNAS 118, e2107934118 (2021).
 https://doi.org/10.1073/pnas.2107934118

Eric D. Crandall, Rachel H. Toczydlowski, Libby Liggins, Ann E. Holmes, Maryam Ghoojaei, Michelle R. Gaither, Briana E. Wham, Andrea L. Pritt, Cory Noble, Tanner J. Anderson, Randi L. Barton, Justin T. Berg, Sofia G. Beskid, Alonso Delgado, Emily Farrell, Nan Himmelsbach, Samantha R. Queeno, Thienthanh Trinh, Courtney Weyand, Andrew Bentley, John Deck, Cynthia Riginos, Gideon S. Bradburd, Robert J. Toonen. **[Importance of timely metadata curation to the global surveillance of genetic diversity](https://conbio.onlinelibrary.wiley.com/doi/full/10.1111/cobi.14061?af=R)** Conservation Biology, 2023.

### Metaanalysis of gradient variation

Molly Albecker, Geoff Trussell, Katie Lotterhos

**Goals:** The goal of this working grooup is to quantify co-gradient and counter-gradient variation (covariance between phenotypic and genetic effects on phenotypes) from common garden and reciprocal transplant experiments, and conduct a metaanalysis to determine how common are patterns of gradient variation in the literature, compared to genetic x environment interactions.

Product: [A novel analytical framework to quantify co-gradient and countergradient variation] (https://onlinelibrary.wiley.com/doi/full/10.1111/ele.14020) Published in Ecology Letters.

## Working groups formed in 2021

### Temporal Genomics

René Clark, Brendan Reid, Anthony Snead, Katrina Catalano, Kyra Fitz, Eric Garcia, Kyle Jaynes, Allyson Salazar Sawkins, John Whalen, and Malin Pinsky

**Goals:** The goal of this working group is to write a comprehensive literature review synthesizing the effects of methodology on our ability to detect recent evolutionary changes across taxa and habitats. Our aim is to not only provide a comprehensive evaluation of past temporal genomics work but to also create a decision framework to help guide the design of future studies in hopes of accelerating the use of temporal genomics in understanding evolutionary responses to change across systems, taxa and time. We will pair this literature review with a series of presentations from researchers currently engaged in temporal genomics projects with the hope of providing scientists at all career stages an opportunity to discuss their research, network, and spark exciting new collaborations.

Website: [Temporal Genomics](https://tempgenomics-rcn.github.io/website/)

### Do spatial and genomic estimates of connectivity agree?

Alli Cramer, Jennifer Hoey, Eric Palkovacs, Carlos Garza, Cynthia Valadon, Remy Gatins, and Jason Toy

**Goals:** Dispersal is an important process both on intra- and inter-generational timescales, but understanding how individual movement translates into genetic population structure and connectivity remains an active area of research. The goal of the working group is to bring researchers in the fields of population genomics and spatial ecology together to understand  how individual dispersal patterns are reflected in genomic population structure. We will pair genomic population structure data with detailed physical tagging data for a few species representing a diversity of movement ecologies. Using the SLiM framework, we will develop simulations that approximate species’ movement ecologies based on the physical tagging and genetic data in order to understand the predicted genomic population structure resulting from each dispersal scenario. We will then compare observed population structure with simulation results, and iteratively vary movement attributes to see how they impact agreement between simulations and measured data. To bridge the gap between population connectivity estimates from the perspectives of population genomics and spatial ecology we must reconcile different definitions of connectivity and the influence of different data types on common connectivity estimations. Identifying a common framework for unifying these branches of population research will allow for increased syntheses of disparate data types and ultimately an increased understanding of population dynamics and the distribution of genetic diversity across the sea.

### MarineOmics

Katherine Silliman, Sam Bogan, Hanny Rivera, Joe McGirr, Megan Guidry, Natalie Ameral, Sara Schaal, Alan Downey-Wall, Dani Davenport, Sara Wuitchik, Nicolas Lou, Joanna Griffiths, Andrew Whitehead, Jon Puritz
Undergraduates: Jessica (Jecy) Klinkam (College of William and Mary)
Christine (Chris) Mantegna (University of Washington)
Kaitlin Macaranas (California State University-Bakersfield)

**Goals:** Population genomic and functional genomic methods are driving advancements in our understanding of population dynamics, physiology, and the evolution of marine populations and species. As next generation sequencing applications and bioinformatic analyses progress, it
is essential to evaluate, update, and guide their use in a manner that realizes the unique contexts and obstacles within the systems these tools are applied. The goal of the MarineOmics working group is to formulate sets of recommended practices for reproducible
and robust genomic research in marine science. These guidelines will be disseminated through an easily accessed, dynamic website resource. The working group will also conduct and publish a qualitative review of reproducibility, standardization, and accessibility
of data/code in genomic studies of environmental adaptation in marine populations. Funds from the RCN-ECS will support three undergraduate research interns, with the goal of broadening participation in the fields of marine science and molecular ecology among early career researchers.

website: [MarineOmics](https://marineomics.github.io/)

### Seascape Genomics of North Pacific Forage Fishes

Savannah LaBua, Laura Timm, Anna Rix, Nicholas Tucker, Jessica Glass, Andres Lopez, Kevin Boswell.

**Goals:** We aim to connect the worlds of fisheries research and evolutionary genomics by applying and broadly testing evolutionary concepts in the context of vast datasets and large patterns of species composition and abundances that have been carefully tracked and examined in fisheries science. We aim to produce interdisciplinary assessments between genomics, physical oceanography, bioinformatics, ecology, and fisheries management to address large-scale ecosystem and population dynamics as climate conditions continue to change.

### Bioinformatics workshop as a gateway to kelp and urchin genomics

Serena Caplins, Maddie Armstrong, Rachael Bay, Vanessa Guerra, Michael Dawson, Patrick Krug, and Joshua Melendez ++ a cohort of undergraduates who will be recruited from the workshop

**Goals:** Modern evolutionary genetics relies on the manipulation of increasingly large datasets necessitating training in computer programming. The utility of this training is not always clear to marine science majors and opportunities for training in computer science are often inequitable. Our working group will train undergraduates to do bioinformatics over a 10-week long workshop, which will showcase clear and exciting application of bioinformatic tools to the most compelling questions in marine evolutionary biology. We will recruit several students from the workshop to apply their training in a novel population genetic analysis of two iconic species from the California coast, purple urchin and giant kelp. We will make course notes that are easy to understand and implement from the workshop available to the RCN community and the broader scientific community. 

website: [Bioinformatics for Marine Science](https://baylab.github.io/MarineGenomics/week-1-welcome.html)


### Blinded by the Wright? Prevalence of adaptive divergence in phenotypic traits at microgeographic scales

Dan Bolnick, Wes Dowd, Atsushi Fujimura, Cameron Ghalambor, Torrance Hanley*, Randall Hughes, Erik Sotka, Phil Yund

**Goals:** Evolutionary biologists are increasingly aware of the potential for organisms to exhibit putatively adaptive differentiation over surprisingly small spatial scales, even those well within the dispersal range of a species. For example, even marine species with long-lived, highly dispersive, pelagic larvae may exhibit fine-grained local adaptation. These observations appear to contradict conclusions of the classic Wright migration-selection balance model. Motivated by a need to more comprehensively evaluate the prevalence of this phenomenon, we are conducting a meta-analysis with two objectives: 1. to quantify the strength of trait-environment correlations across spatial scales, and 2. to assess whether these scale-dependent patterns might vary across taxa (e.g., plant, invertebrate) or ecosystem (e.g., terrestrial, marine). We are focusing on studies that included common garden and/or reciprocal transplant evaluations of phenotypic traits across environments. If prevalent, microgeographic scales of adaptation demand refined approaches to evolutionary biology and conservation biology, with possible implications for a variety of other fields.

